store/sqlstore/channel_store.go

```diff
@@ -1730,7 +1730,7 @@ func (s SqlChannelStore) AutocompleteInTeamForSearch(teamId string, userId strin
 			JOIN
 				ChannelMembers AS CM ON CM.ChannelId = C.Id
 			WHERE
-			    C.TeamId = :TeamId
+			    (C.TeamId = :TeamId OR C.TeamId = '')
```

Comment:
Can you update the store unit tests to verify this new behaviour?

Reply:
Done!